src/content/blog/2025-01-31-ilp-packet-lifecycle.mdx

```diff
@@ -0,0 +1,88 @@
+
+A prospective debitor will use this standard to set payment terms by first creating an _incoming payment_ on the Rafiki instance hosting the receiving wallet address, then acquiring a _quote_ from the Rafiki instance hosting the sending wallet address.
+Finally, it creates an _outgoing payment_ against the sending wallet address by acquiring consent from its owner to charge them with particular terms such as its amount and its currency.
+Once that is complete, these Rafiki instances can begin to fulfill the terms of this payment using Interledger.
+
+On a more technical point, ILP packets may be involved before an outgoing payment is even created. During the acqusition of a _quote_ from the sending Rafiki instance, it will send a ILP packet accross the network to the destination. This packet will need to traverse each hop on the network as each node's fees and rates won't be known to the sender without doing so.
+
+_For a more on Open Payments, consider reading the [Simple Open Payments Guide](https://interledger.org/developers/blog/simple-open-payments-guide/) on this blog._
+
+### Preparing the Packet
+Now that the sending Rafiki instance has an outgoing payment created against one of its wallet addresses, it can pass the terms of that payment into an Interledger payment. First, it uses the [Simple Payment Setup Protocol](https://github.com/interledger/rfcs/blob/main/0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md) to acquire an ILP address for the receiver, and a shared secret that is used to generate a [_condition_](https://github.com/interledger/rfcs/blob/main/0029-stream/0029-stream.md).
+A condition is a hash that can only be generated with a secret that is available only to the receiver and the sender.
+Using the receiving ILP address, the condition, the receiver, and send amount described in the outgoing payment, Rafiki creates a connection over ILP to the receiver using [STREAM](https://github.com/interledger/rfcs/blob/main/0029-stream/0029-stream.md).
+STREAM is a transport-layer protocol for Interledger that is responsible for breaking a payment down into the aforementioned packets and sends them _en masse_ until the payment is fulfilled.
+
+For each packet, STREAM handles them using a two-phase execution process that is similar to the request-response process that HTTP packets sent over the internet use.
+
+<LargeImg
+  src="/developers/img/blog/2025-01-31/2p-transfer.png"
+  alt="Two-Phase Transfer Diagram"
+/>
+
+In the first phase, the sender creates packets known as ["prepare" packets](https://github.com/interledger/rfcs/blob/main/0027-interledger-protocol-4/0027-interledger-protocol-4.md#ilp-prepare), which each contain some fraction of the total send amount, the condition generated using SPSP, the final destination of the packet in the network, and an expiration time for the packet.
```

Comment:
```suggestion
In the first phase, the sender creates packets known as ["prepare" packets](https://github.com/interledger/rfcs/blob/main/0027-interledger-protocol-4/0027-interledger-protocol-4.md#ilp-prepare), which each contain some fraction of the total send amount, the condition generated using SPSP, the final destination of the packet in the network, and an expiration time for the packet.
```
do we want to mention that the fraction is something configured by the peers?

Reply:
I think the overall story remains coherent without including this detail. I can add a further reading link to https://interledger.org/developers/rfcs/peering-clearing-settling/ at the end, perhaps.